source/uwp/AdaptiveCardTestApp/Expected/v1.0.Elements.Input.ToggleToJson.json

```diff
@@ -1 +1,5 @@
-{"actions":[{"title":"OK","type":"Action.Submit"}],"body":[{"text":"Toggle Input","type":"TextBlock"},{"id":"acceptTerms","isRequired":false,"title":"I accept the terms and agreements","type":"Input.Toggle","value":"true","valueOff":"false","valueOn":"true"}],"type":"AdaptiveCard","version":"1.0"}
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
This file didn't merge correctly

Reply:
Tx

---
In reply to: [308932979](https://github.com/microsoft/AdaptiveCards/pull/3278#discussion_r308932979) [](ancestors = 308932979)